.circleci/config.yml

```diff
@@ -38,6 +43,9 @@ jobs:
       - restore_cache:
           keys:
             - java9-gradle-{{ checksum "build.gradle" }}
+      - run: sudo apt-get update && sudo apt-get install -y make
```

Comment:
The make command is not installed in openjdk9 image. However it is present in openjdk8.

Reply:
Please add a comment in the file.

Reply:
I think the install commands seems self-explanatory tha make is missing. But I can add a comment too.